manager-dashboard/app/views/NewTutorial/ScenarioPageInput/ChangeDetectionGeoJsonPreview/styles.css

```diff
@@ -0,0 +1,19 @@
+.change-detection-geo-json-preview {
+    .content {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        gap: var(--spacing-extra-large);
+
+        .map-preview {
+            align-self: center;
+            width: var(--height-mobile-preview-change-detection-content);
+            height: var(--height-mobile-preview-change-detection-content);
+        }
+    }
+
+    .popup {
+        top: 50%!important;
```

Comment:
@frozenhelium do we need to use !important here?

Reply:
We can have this option in the mobile preview iteself